server/store.go

```diff
@@ -216,41 +214,41 @@ func (p *Plugin) popFromOrderForUser(userID string) error {
 	return errors.New("unable to store order")
 }
 
-func (p *Plugin) storeOrder(userID string, order []string, originalJSONOrder []byte) (bool, error) {
+func (p *Plugin) storeList(userID string, listID string, order []*OrderElement, originalJSONOrder []byte) (bool, error) {
```

Comment:
1/5 s/storeList/saveList/ - I just learned myself that Get/Sabe is the de-facto convention in the mattermost code base.